ladybug/datacollection.py

```diff
@@ -14,7 +14,7 @@
 class DataCollection(object):
     """A list of data with a header."""
 
-    __slots__ = ('_header', '_data')
+    __slots__ = ('_header', '_data', 'highest_values', 'highest_values_index')
```

Comment:
I commented down there. Let's not add new properties to class as they are not really neccassary for all the instances of the class.